lib/src/testing/webcrypto/digest.dart

```diff
@@ -63,7 +63,7 @@ List<({String name, Future<void> Function() test})> tests() {
   });
 
   test('SHA-1: libsum', () async {
-    const h = Hash.sha1;
+    final h = Hash.sha1;
```

Comment:
We won't need this change if we make `Hash.sha1` `const` again. See other comment.

Reply:
Reverted 🫡